inspirehep/factory.py

```diff
@@ -55,7 +55,7 @@
 """
 
 create_api = create_app_factory(
-    'inspirehep',
```

Comment:
If I remember correctly, I once tried to change this name, but the consequence is that the configuration file will then not be found by the API application.

In `var/instance/inspirehep.cfg` (the name has to coincide with the app name) so this would not work for us.